frontend/packages/core/src/AppLayout/logo.tsx

```diff
@@ -11,8 +11,9 @@ const rotate = keyframes`
 `;
 
 const StyledSvg = styled.svg`
-  height: 40px;
-  width: 40px;
+  height: 32px;
```

Comment:
figma designs specify 32px for width and height but it looks smaller in storybook than it does on the design.
here is what it looks like at the original size (40px)
<img width="580" alt="Screen Shot 2020-11-19 at 6 12 34 PM" src="https://user-images.githubusercontent.com/39421794/99735646-f8a3ac80-2a92-11eb-83d2-a8f09b6aa007.png">